docs/quickstart.md

```diff
@@ -23,6 +23,8 @@ Base.
         -w /input/scientists.wordsfile.tsv \
         -d /input/scientists.docsfile.tsv
     qlever@xyz:/app$ exit
+    
+**Note for Mac users.** Having errors building an index follow [troubleshooting](./troubleshooting.md#note-for-mac-users).
```

Comment:
This doesn't sound grammatically correct and I think it's a bit confusing calling "Docker on macOS" "Mac users". Maybe:
```
**Note:** If you experience an error during index building check the [troubleshooting](./troubleshooting.md) document for known issues such as problems with Docker on Mac.
```

Reply:
The Mac users should be fine. The problem is not in the docker itself but in APFS which is all Macs feature, so this should be also a problem running on native Mac env. 
I can recheck this, but it can take some time run everything on native Mac. Do you think this is useful to try to run it on Mac?

Reply:
I don't think it makes sense for us to invest too much time in porting to macOS. I also think that Docker for Mac is the only variant that's really relevant here as it should work in principle. Arguing whether it would also be a problem when running directly on macOS is kind of moot since that's really not supported. But yeah I did mention file systems as one could indeed get the same problem on Linux when using e.g. a NTFS disk shared between Windows and Linux. So lets make this about the file systems and not about using Mac.

Reply:
agree